ogre/src/OgreLidarVisual.cc

```diff
@@ -239,9 +284,31 @@ void OgreLidarVisual::Update()
       mv = std::dynamic_pointer_cast<Ogre::MovableObject>(line);
       this->Node()->attachObject(mv.get());
       this->dataPtr->rayLines.push_back(line);
+      }
+      this->dataPtr->deadZoneRayFans[j]->SetPoint(0, this->offset.Pos());
     }
-    this->dataPtr->deadZoneRayFans[j]->SetPoint(0, this->offset.Pos());
 
+    else if (this->dataPtr->lidarVisType ==
+                  LidarVisualType::LVT_POINTS)
+    {
+      if (j+1 > this->dataPtr->points.size())
+      {
+        std::shared_ptr<OgreDynamicLines> line =
+                  std::shared_ptr<OgreDynamicLines>(
+                              new OgreDynamicLines(MT_POINTS));
+
+        #if (OGRE_VERSION <= ((1 << 16) | (10 << 8) | 7))
+              line->setMaterial("Default/White");
+        #else
+              line->setMaterial(
+                  this->Scene()->Material("Default/White")->Clone());
```

Comment:
same comment as before. Need to use `Ogre::MaterialManager`

Reply:
and can we keep the color consistent as other visualization type? i.e. blue. Note that we should also make the material configurable by the user by overriding the `SetMaterial` function. This can come later in a separate PR.